.github/codespell_ignore_words_file.txt

```diff
@@ -0,0 +1,23 @@
+aci
```

Comment:
Where is this used? I can not find it in the sources anywhere?
Does this file support some kind of comments to give some comments why the unclear words are here?

Reply:
I see obvious groups of words in other languages (Portuguese?, German, Russian), which look on the first sight as a typo.

Reply:
https://github.com/OpenSC/OpenSC/blob/7a6032d3/src/libopensc/card-setcos.c#L439

Reply:
Looks like  https://github.com/OpenSC/OpenSC/blob/7a6032d3/src/libopensc/card-setcos.c#L439 should be "ACL" not "ACI".  So aci not needed in codespell_ignore_words_file.txt

Reply:
Good catch. Indeed sounds like a typo from the context to me too

Reply:
@DimitriPapadopoulos can you fix also the ACI typo to ACL? I think this will not be either in the default dictionary either.

Reply:
Fixed.